src/node-explorer-provider.ts

```diff
@@ -266,9 +279,9 @@ export class FileExplorer extends vscode.TreeItem {
         arguments: [this.uri],
       };
     }
-  }
 
-  contextValue = 'file-explorer-item';
+    this.contextValue = `file-explorer-item${context ? '-' : ''}${context}`;
```

Comment:
For my own education, what's the purposes of setting `this.contextValue` for vscode?

Reply:
It allows us to use pattern matching for the context menus as we do here: https://github.com/tailscale-dev/vscode-tailscale/pull/119/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R154